src/api-reference/common/event-pubsub/travel-search-event.markdown

```diff
@@ -107,7 +107,8 @@ Name|Type|Format|Description
 Name|Type|Format|Description
 ---|---|---|---
 `companyId`|`String`|GUID|Uniquely identifies the company of the traveler.
-`userId`|`String`|GUID|Uniquely identifies the user performing the search. *NOTE:* In the event travel is booked by an arranger, this is the traveler. In cases where the individual is booking on behalf of a guest, this is the user performing the search.
+`userId`|`String`|GUID|Uniquely identifies the user performing the search. *NOTE:* In cases where the individual is booking on behalf of a guest, this is the user performing the search (e.g. the arranger).
+Unlike Air, Hotel will not return arranger information.
```

Comment:
The hard line break doesn't work within a table structure. It needs to be in paragraph form and, on rare occasions, if you must, you can use `<br>` to make soft line breaks within the table.

Reply:
Thanks! Fixed.